src/Games/brain-cals.js

```diff
@@ -0,0 +1,31 @@
+import getRandomNumber from '../randomCount.js';
+import theCoreOfGames from '../index.js';
+
+const brainRulls = () => {
+  const descriptionOfGame = 'What is the result of the expression?';
+
+  const randomNumberOne = getRandomNumber(1, 10);
+  const randomNumberTwo = getRandomNumber(1, 10);
+  const operator = ['+', '-', '*'];
+  const randomOperator = getRandomNumber(0, 2);
+  const randomNumber = `${randomNumberOne} ${operator[randomOperator]} ${randomNumberTwo}`;
+
+  let correctAnswer;
```

Comment:
I think the logic for getting the correct answer should be moved to a separate function. getResultExpression(num1, num2, operation)?

Reply:
added